install_ffmpeg.sh

```diff
@@ -76,8 +76,8 @@ if [ ! -e "$ROOT/x264" ]; then
 fi
 
 if [[ $(uname) == "Linux" && $BUILD_TAGS == *"debug-video"* ]]; then
+  sudo apt-get install -y libnuma-dev cmake
```

Comment:
Is the reason for moving this install step up here because we need to make sure these deps are installed even if x265 is already installed i.e. if the conditional on L80 evaluates to false?

Reply:
Yes - in CircleCI we have x265 folder cached, so `libnuma` is not installed (that's also why other PRs in LPMS fails now)